python/pythonlab/neighborhood/neighborhood/painter.py

```diff
@@ -190,3 +187,30 @@ def set_paint(self, paint):
       paint (int): The amount of paint that should be in the painter's bucket.
     """
     self.remaining_paint = paint
+
+  def _get_initialization_message(self, x, y, direction, paint):
+    detail = {
+                'id': f"painter-{Painter.last_id}",
```

Comment:
should we pass this in so we don't have it defined twice? (`self.id` doesn't work here I'm guessing?)

Reply:
Actually `self.id` does work so will go with that. Thanks!